doc/api/util.md

```diff
@@ -10,6 +10,67 @@ module developers as well. It can be accessed using:
 const util = require('util');
 ```
 
+## util.callbackify(original)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `original` {Function} An `async` function
+* Returns: {Function} a callback style function
+
+This method converts Promise returning API endpoints to ones that use callbacks.
+`callbackify` takes an `async function` (or a function that returns a Promise)
+and returns a function following the common Node.js callback style, i.e. taking
+a `(err, val) => ...` callback as the last argument. In the callback, the first
+argument will be the rejection reason (or `null` if the Promise resolved), and
+the second argument will be the resolved value.
+
+For example:
+
+```js
+const util = require('util');
+
+async function fn() {
+  return await Promise.resolve('hello world');
+}
+const callbackFunction = util.callbackify(fn);
+
+callbackFunction((err, ret) => {
+  if (err) throw err;
+  console.log(ret);
+});
+```
+
+Will print something like:
+
+```txt
+hello world
+```
+
+*Note*:
+
+* Like with most callback style functions, the callback is executed in an
+async context (i.e. having a limited stacktrace). If the callback throws, the
```

Comment:
Can you shorten the first sentence to:


The callback is executed asynchronously, and will have a limited stack trace.